modules/feature-management/src/Volo.Abp.FeatureManagement.Application/Volo/Abp/FeatureManagement/FeatureAppService.cs

```diff
@@ -65,6 +66,51 @@ public virtual async Task UpdateAsync([NotNull] string providerName, [NotNull] s
             }
         }
 
+        [Authorize(FeatureManagementPermissions.ManageHostFeatures)]
+        public async Task<FeatureListDto> GetHostAsync()
```

Comment:
I suppose no need to these duplicated methods if we make providerKey nullable in the get/update methods.
For the authorization part, we can reuse the CheckProviderPolicy system, by defining the policy at `Options.ProviderPolicies`.